src/surfaces/modal/modal.jsx

```diff
@@ -242,11 +137,26 @@ class Modal extends React.Component {
             this.setState({
                 isOpen,
             }, () => {
-                this._onOpen();
+                this.onOpen();
             });
         }
     }
 
+    componentDidUpdate(prevProps, prevState) {
+        if (!prevProps.isOpen && this.props.isOpen) {
+            this.setState({
+                isOpen: this.props.isOpen,
```

Comment:
Is it worth destructuring from `this.props` here to appease ESLINT?

Reply:
Probably, but i already did this in v.10. I am wondering if we can get some of that work into v.9 one day.

Reply:
Oh, nm, I realized this was the one component i tried to get the red out of my face. I destructured.

Reply:
Lookin' sharp.  Resolving.